### ISSUE TITLE
Auto ReflowLayout on node size change?

### ISSUE THREAD

**Issue by kabriel** [IjzerenHein/famous-flex#10]
First let me say that what you have done so far is awesome. This fits very well into what I've already been doing, and you have made a great framework out of it.

I have some nodes/views that determine their size dynamically, and that size can change over time. If I use/build a layout that uses 'true' sized nodes, such as with NavBar, it lays them out initially, but if their size changes, it does not re-layout (size is not requested again until reflow). If I detect the change and call reflowLayout(), that works, but is pretty messy and breaks a lot of abstractions. 

What do you think about adding (maybe as an option, so it can be turned off if not needed) a check (with a cache) to see if any of the current targets have changed size? If any have, then it would auto reflow.

**Comment by gadicc** [comment]
Hey @IjzerenHein, [this post](https://github.com/gadicc/meteor-famous-views/issues/186#issuecomment-66226659) might interest you.  I added a kind of similar option in famous-views, I called it "watchSize" (for true-sized surfaces)... initially using MutationObserver and then after the linked comment above, switched to his library using underscroll/overscroll events.  I also make clear in the docs that it's still better to avoid it if possible, but I definitely agree it's better that we (lib providers) make this option available cleanly to other devs.  Nothing urgent but just so you're aware of it, since our users had similar issues.
